Add up next icon to episode chip

mchowning opened the pull request:
## Description
This adds an up-next icon to the `EpisodeChip`.

In addition, this updates the episode chip to vertically center the podcast image when there are no icons present. Unfortunately, you can't test this currently because the only screens using this chip right now are the Up Next screen and the Downloads screen and I certainly hope that any podcast on one of those screens should either be in the Up Next or downloaded. 😄 I've included a screenshot below though so you can see what it would look like.

> **Note**
> This PR builds on https://github.com/Automattic/pocket-casts-android/pull/909.

## Testing Instructions
1. Go to the Up Next queue or the Downloads page
2. Observe that the up-next icon and download icons are presented when appropriate

## Screenshots or Screencast 

| Both icons | Up Next only | Downloaded only | No icons |
| --- | --- | --- | --- |
| ![image](https://user-images.githubusercontent.com/4656348/234697996-3b63eeae-5f9f-4c58-b645-5a80b62e1586.png) | ![image](https://user-images.githubusercontent.com/4656348/234698045-32318683-265b-44a9-840f-86ad3fd6b439.png) | ![image](https://user-images.githubusercontent.com/4656348/234698159-c0903e68-917a-4116-9b85-1b6dd3fadfdb.png) | ![image](https://user-images.githubusercontent.com/4656348/234698661-4165d348-7278-40dd-9bbf-1299a8b4e2fa.png) |

## Checklist
- [X] If this is a user-facing change, I have added an entry in CHANGELOG.md
- [X] I have considered whether it makes sense to add tests for my changes
- [X] All strings that need to be localized are in `modules/services/localization/src/main/res/values/strings.xml`
- [X] Any jetpack compose components I added or changed are covered by compose previews

ashiagr commented:
Looks good, just that up-next icon appears a little bit higher 

App

<img width=200 src="https://user-images.githubusercontent.com/1405144/234796623-0322015b-76ee-4dca-a5b2-1dd0f520014e.png"/>

Figma 

<img width="200" alt="Screenshot 2023-04-27 at 1 22 31 PM" src="https://user-images.githubusercontent.com/1405144/234796723-eacaaa77-dc65-40c2-a383-04b790146502.png">

mchowning commented:
> up-next icon appears a little bit higher

Thanks for calling that out. I'm not sure why the up next icon in the project isn't centered vertically, so I updated it from FIgma. The new icon uses a slightly thicker strokes and generally has a slightly different style, so I also updated the download icon so they both have the same style ([a894914](https://github.com/Automattic/pocket-casts-android/pull/914/commits/a894914b4c76bf7d3c9a57de7149e2b237b4673e)). Note that this change also updates the icons in the phone app.

| Before | Now |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4656348/234697996-3b63eeae-5f9f-4c58-b645-5a80b62e1586.png) | ![image](https://user-images.githubusercontent.com/4656348/234846734-7e3a8cac-196f-4907-b2fe-42740141fc2e.png) |
